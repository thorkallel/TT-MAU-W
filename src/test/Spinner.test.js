import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

import Spinner from "../components/Spinner";

describe("Pruebas en <Spinner />", () => {
  test("debe mostrar <Spinner /> correctamente", () => {

    const wrapper = shallow(
      <Spinner />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
