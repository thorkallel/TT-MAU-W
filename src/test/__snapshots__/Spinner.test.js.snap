// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Spinner /> debe mostrar <Spinner /> correctamente 1`] = `
<div
  className="sk-chase"
>
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
</div>
`;
